backend/api/auth_endpoints.py

```
@@ -0,0 +1,281 @@
+        )
+
+    except Exception as e:
+        logger.error(f"Error during social login: {e}")
+        raise HTTPException(status_code=500, detail="Social authentication service error")
+
+
+@auth_router.get("/session", response_model=Dict[str, Any])
+async def get_session(request: Request) -> Dict[str, Any]:
+    """
+    Get current session information in NextAuth compatible format.
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)